apps/src/templates/rubrics/LearningGoals.jsx

```diff
@@ -221,6 +224,13 @@ export default function LearningGoals({
 
   const aiEvalInfo = getAiInfo(learningGoals[currentLearningGoal].id);
 
+  let aiConfidence;
+  if (aiEvalInfo) {
+    aiConfidence = aiEvalInfo.showExactMatch
+      ? aiEvalInfo.aiConfidenceExactMatch
+      : aiEvalInfo.aiConfidencePassFail;
+  }
```

Comment:
this change updates aiConfidence to depend on whether we are showing pass-fail or exact-match AI scores. all uses of aiConfidence in client code should now refer to this computed value, not the `ai_confidence` value on the server which is actually the pass-fail confidence level.